### PR TITLE
Improve user experience ztl

### PR DESCRIPTION
**Pull request by zhangtianli2006**

### CODE REVIEW

**Comment by zhangtianli2006** [comment]
Since you're all at home, I took some screen shots  to show.

basically, I made the problem edit page better and fixed the user info page in mobile.
and the tool bar in the problem page now will be activate when `active`, rather than `hover`.

**Comment by szdytom** [comment]
excellent

**Comment by szdytom** [comment]
@gzezFISHER would you mind a look?

**Comment by zhangtianli2006** [comment]
come on, why it has to be FISHER

**Comment by zhangtianli2006** [comment]
？？

**Comment by zhangtianli2006** [comment]
I think I'm the only one with a BE server at home...
I've tested on my phone as well

**Comment by szdytom** [comment]
> 
> 
> I think I'm the only one with a BE server at home...
> I've tested on my phone as well

of course I have BE server...

File: src/sfconfig.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export default {
   api: {
-    server: 'http://172.32.1.144:8000/api'
+    server: 'http://localhost:8000/api'
```
Comment (szdytom) [inline src/sfconfig.js:3]: Maybe we should get this file out of git repo...

Comment (zhangtianli2006) [reply src/sfconfig.js:3]: it's up to you

**Pull request merged**